gostrufig.go

```diff
@@ -23,45 +23,40 @@ const CFGDEFAULT string = `cfg-def`
 // When retrieving this config for an application called "testApp", it can be found at "/testApp/brockwood/123/"
 const CFGNAMESPACE string = `cfg-ns`
 
+var gsfdriver Driver
+
 // Configuration response codes
 const (
-	CONFIGFOUND    int = 100
-	CONFIGNOTFOUND int = 200
+	CONFIGNOTFOUND int = 100
+	CONFIGFOUND    int = 200
 )
 
-type GoStruFig struct {
+type Gostrufig struct {
 	appName        string
-	driver         Driver
 	driverLocation string
 	config         interface{}
 }
 
 // GetGoStruFig returns an instace of the configuration with the given name, driver and configuration URL.
-func GetGoStruFig(appname, drivername, location string) GoStruFig {
```

Comment:
What is the thought process behind passing in a string that references a driver type vs passing in an instance of a struct that implements a Driver interface?